### ISSUE TITLE
Can't get build to run on os x 12.2.1 Monterey

### ISSUE THREAD

**Issue by kelly-tock** [javierbrea/pnpm-nx-monorepo-example#6]
getting this:
![image](https://user-images.githubusercontent.com/39061683/168139415-3288468a-11fa-4083-a736-10d41ee14ad3.png)

on this repo as well as my own when following along. guessing i'm missing something in terms of how the command ends up being run with nx?  great article!

**Comment by javierbrea** [comment]
Hi @kelly-tock , 

It seems that it is related to the Pnpm version. I have reproduced the issue upgrading it to the latest version. From version 7, a [`--` is automatically added when running pnpm scripts](https://github.com/pnpm/pnpm/releases/tag/v7.0.0), and this produces an error due to the `exit 0` script in the `sum-two` package.

I have removed the `&& exit 0` from the example script and now it works again. I will remove it also from the article examples. Thank you very much for your feedback.
